src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -2306,6 +2306,23 @@ export class ZoneServer2016 extends EventEmitter {
 
     const sourceEntity = this.getEntity(npcTriggered),
       sourceIsVehicle = sourceEntity instanceof Vehicle2016;
+
+    // explode nearby explosives first
```

Comment:
but doesn't it allow to kill with an ied in PVE ?

Reply:
No sir 👍 All is good for the PvE-er's